moncli/entities/column_value/simple.py

```diff
@@ -211,6 +211,55 @@ def _convert(self, value):
     def _format(self):
         return {'text': self._value}
 
+class LocationValue(ComplexNullValue):
+    """A location column value."""
+
+    native_type = Location
+    allow_casts = (dict, str)
+
+    def _convert(self, value):
+        return Location(lat=value['lat'],lng=value['lng'],address=value['address'])
+
+    def _cast(self, value):
+        return_value = None
+        if isinstance(value,dict):
+            try:
+                lat = float(value['lat']) if value['lat'] else 0.0
+                lng = float(value['lng']) if value['lng'] else 0.0
+                address = value['address'] if value['address'] else None
+                return_value = Location(lat=lat,lng=lng,address=address)
+            except KeyError:
+                raise ColumnValueError(
+                    'invalid_location_data',
+                    self.id,
+                    'Unable to convert "{}" to Location value.'.format(value)
+                )
+        elif isinstance(value,str):
```

Comment:
This does not look entirely correct here.

The following code does what we are trying to do a little better.

```python
elif isinstance(value, str):
    values = value.split(' ', 3) # I really like this max split feature!
    if len(values) < 2: # The split must contain at least two items for lat and lng
        raise ColumnValueError(
            'invalid_location_data',
            self.id,
            'Both a latitude and longitude are required for a Location value.')
    try:
        location = Location(float(values[0]), float(values[1])) # The lat and lng values must be cast into floats
        try:
            location.address = values[2] 
        except IndexError:
            pass # We don't care if there is no value for address
        return location
    except ValueError:
         raise ColumnValueError(
             'invalid_location_data',
             self.id,
             'Input str value "{}" contains invalid coordinates.'.format(value))
```